AENS.md

```diff
@@ -268,27 +268,30 @@ commitment := Hash(NameHash(name) + name_salt)
  ---------------- ----
 | name           | 63 |
  ---------------- ----
-| fee            | 32 |
+| rent           | 32 |
  ------------- -- ----
 | name_salt      | 32 |
  ---------------- ----
 ```
 
 Flow for a user:
 
-1. (optional) wait `n` blocks, s.t. that the block including the `pre-claim` cannot be reversed whp
+1. (optional) wait `n` blocks, s.t. that the block including the `pre-claim` cannot be reversed with high probability
 2. send `claim` transaction to reveal name and pay the associated fee
 
-If the time delta of `pre-claim` and `claim` is bigger than 300 blocks,
+If the time delta of `pre-claim` and `claim` is bigger than `floor(60 * 24 * 2/key_block_interval)` blocks,
 then the `claim` MUST be rejected.
 
-The `claim` transaction MUST be signed by the same private key as a
-`pre-claim` transaction containing a commitment to the name and nonce.
+The `claim` transaction MUST be signed by the same private key as a `pre-claim`
+transaction containing a commitment to the name and nonce.
 
 A `claim` transaction MUST NOT be in included in the same block as its
 `pre-claim`.
 
-Note that the fee here is a distinct fee from the normal transaction fee.
+`rent` is the reoccurring fee that has to be paid for names. Our first rent
+model is a naïve one, that just uses a flat fee. Our initial target is 4 aeons
```

Comment:
Do we use *aeon* as a name of the currency? Don't we rather use _AE tokens_, instead? Main concern is that _Aeon_ is already used as a currency name...
Same comment for the line bellow.